app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -193,4 +188,36 @@ public void toggleLogging(MenuItem item) {
         }
 
     }
+
+    public void displayStaticData() {
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
+        String syncConnPref = preferences.getString(SettingsActivity.KEY_PREF_SYNC_CONN, "");
+
+        TextView tvStaticDataName = ((TextView) findViewById(R.id.tvStaticDataName));
+        TextView tvStaticDataValue = ((TextView) findViewById(R.id.tvStaticDataValue));
+
+        String tvTextName = "";
+        String tvTextValue = "";
+
+        if (preferences.getBoolean("sEngineTemp", false)) {
+            //TODO Read from database
+            tvTextName +="Engine Temperature:";
+            tvTextValue += "80 F ";
+        }
+
+        if (preferences.getBoolean("sMPG", false)) {
+            //TODO Read from database
+            tvTextName += "\nCurrent Miles Per Gallon:";
```

Comment:
Instead of one TextView with multiple newlines, what if we added new TextViews dynamically? Seems more appropriate to me.